rpcd/playbooks/roles/kibana/templates/000-kibana.conf

```diff
@@ -21,25 +21,19 @@
     SSLCipherSuite {{ kibana_ssl_cipher_suite }}
     SetEnvIf User-Agent ".*MSIE.*" nokeepalive ssl-unclean-shutdown
 
-    DocumentRoot {{ kibana_root }}
+    <Proxy *>
+        Order Allow,Deny
+        Allow from all
+        AuthType Basic
+        AuthName Kibana
+        AuthUserFile /etc/apache2/users
+        Require user kibana
+    </Proxy>
 
-    <Directory />
-      Options FollowSymLinks
-      AllowOverride None
-    </Directory>
-
-    # ElasticSearch Reverse Proxy
-    <Location /elasticsearch/>
-        ProxyPass http://{{ elasticsearch_vip }}:{{ elasticsearch_http_port }}/
-        ProxyPassReverse /
-    </Location>
-
-    <Directory {{ kibana_root }}>
-      Options Indexes FollowSymLinks MultiViews
-      AllowOverride AuthConfig
-      AuthType Basic
-      AuthName Kibana
-      AuthUserFile /etc/apache2/users
-      Require user kibana
-    </Directory>
+    ProxyRequests Off
+    ProxyPass / http://127.0.0.1:{{ kibana_app_port }}
```

Comment:
Does this require `proxy` and `proxy_http` Apache modules? Do we actually install those? I didn't think we did.